### ISSUE TITLE
Add contact mail address for accessibility feedback

### ISSUE THREAD

**Issue by Anne-Victoria** [c3wien/pw20-streaming-site#80]
We want to create a mail address where people can let us know when they encounter accessibility issues at privacyweek in general and also with the streaming site specifically. That email address should be placed somewhere on the streaming site.
